compute/src/test/java/org/jclouds/compute/internal/BaseTemplateBuilderLiveTest.java

```diff
@@ -44,6 +49,9 @@
 @Test(groups = "integration,live")
 public abstract class BaseTemplateBuilderLiveTest extends BaseComputeServiceContextLiveTest {
 
+   public static final Object[][] NO_INVOCATIONS = new Object[0][0];
+   public static final Object[][] SINGLE_NO_ARG_INVOCATION = { new Object[0] };
```

Comment:
This is a recurring pattern. Move these constants to the [TestUtils](https://github.com/jclouds/jclouds/blob/master/core/src/test/java/org/jclouds/utils/TestUtils.java) class, and also remove them from the filesystem provider and other classes, so all data providers using this pattern use the same constants.